server/router.ts

```diff
@@ -70,6 +70,10 @@ import createThread from './routes/createThread';
 import editThread from './routes/editThread';
 import deleteThread from './routes/deleteThread';
 import bulkThreads from './routes/bulkThreads';
+import createDraft from './routes/createDraft';
+import deleteDraft from './routes/deleteDraft';
+import editDraft from './routes/editDraft';
+import getDrafts from './routes/getDrafts';
```

Comment:
Thoughts on putting these routes in the same folder, e.g. routes/drafts/createDraft.ts?

Reply:
Aye-aye!